### ISSUE TITLE
Hallucinations from mindbreaker sting make my game crash.

### ISSUE THREAD

**Issue by LufgtHuron** [HippieStationArchive/HippieStation13-1#3134]
Got stung by a ling. Started tripping out and the game froze big time. Had to close dreamseeker in the task manager. This happens every time I get LSD stung. Didn't report it before because I didnt die to it then, now I did.

**Comment by gumner** [comment]
yeah happened to me one time,my game would run at 2 fps and crash every second.

**Comment by ArcLumin** [comment]
I haven't had the issue, might be an issue client side?

**Comment by Spacedong** [comment]
maybe your pc isnt beefy enough

**Comment by LufgtHuron** [comment]
@Spacedong my PC runs all AAA titles on solid 60fps... 

this is a problem with byond/how hallucinations are working and specific graphics hardware probably, sorta like laptop users cant use dedicated cards because then the game doesnt render.
